src/components/views/messages/MessageActionBar.tsx

```diff
@@ -284,6 +303,20 @@ export default class MessageActionBar extends React.PureComponent {
                 toolbarOpts.push(cancelSendingButton);
             }
 
+            if (this.props.isThreadExpanded !== undefined && ReplyThread.hasThreadReply(this.props.mxEvent)) {
+                const expandClassName = classNames({
+                    'mx_MessageActionBar_maskButton': true,
+                    'mx_MessageActionBar_expandMessageButton': this.props.isThreadExpanded === true,
+                    'mx_MessageActionBar_collapseMessageButton': this.props.isThreadExpanded === false,
```

Comment:
```suggestion
                    'mx_MessageActionBar_expandMessageButton': !this.props.isThreadExpanded,
                    'mx_MessageActionBar_collapseMessageButton': this.props.isThreadExpanded,
```

You had the classes flipped and it's more readable this way, imo

Reply:
@SimonBrandner there's a good reason for that, we don't want to show any icons if it's `undefined`, as it can be `true | false | undefined`. We don't really need to show this icon if the text isn't truncated, do we? 😄  
I'll just add a comment there, cause this code looks weird (and it should!)

Reply:
Oh right, I already did the check for it above! Right, fixing! 🤦